src/image/__tests__/ImageAnnotator-test.ts

```diff
@@ -220,13 +234,21 @@ describe('ImageAnnotator', () => {
 
             annotator.annotatedEl = getParent();
             annotator.store.dispatch(fetchAnnotationsAction.fulfilled(payload, 'test', undefined));
+            annotator.store.dispatch(
+                fetchAnnotationsAction.fulfilled({ ...payload, entries: drawings as Annotation[] }, 'test', undefined),
```

Comment:
Does this need to be dispatched twice or can we update the `payload` variable to include the data we need?